workspaces/ui/src/utilities/debug-dump.js

```diff
@@ -0,0 +1,33 @@
+import * as deepCopy from 'deepcopy';
+
+export function debugDump(specService, captureId) {
+  return async function () {
+    const events = await specService.listEvents();
+    const session = await specService.listCapturedSamples(captureId);
+    const sessionCleaned = deepCopy(session);
+
+    console.log('sanitizing data...');
+    sessionCleaned.samples.forEach((i) => {
+      i.request.body.value.asJsonString = null;
+      i.request.body.value.asText = null;
+
+      i.response.body.value.asJsonString = null;
+      i.response.body.value.asText = null;
+    });
+
+    const output = JSON.stringify({
```

Comment:
I'd probably `JSON.stringify(..., null, 2)` so it's more friendly to look at by default

Reply:
The shape hash makes it pretty ugly to look at when formatted. each bit gets it's own line. And it takes up a whole lot more disk space